portiaui/app/instance-initializers/error-handler.js

```diff
@@ -32,20 +45,13 @@ export function initialize(applicationInstance) {
             logErrorStack(err);
             notificationManager.add({
                 title: err.title || 'Unexpected error',
-                message: 'An unexpected error has occurred. Please notify the developers. ' +
-                    'Details have been logged to the console.',
+                message: 'An unexpected error has occurred. ' + instructions,
                 type: 'danger'
             });
         }
     }
 
     Ember.onerror = notifyError;
-
-    applicationInstance.ApplicationRoute = Ember.Route.extend({
```

Comment:
removed since this is causing double logging now. probably ember 2 calls Ember.onerror.